.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Sopel's IRC channel
+    url: https://webchat.freenode.net/#sopel
+    about: "Someone is almost always available to answer questions in #sopel on freenode."
```

Comment:
I like this. Whether you switched to a webchat link because GitHub wouldn't show the IRC one or because it was more noob-friendly, I approve. The channel and network names are still there for people who just want to hop in their existing client.